mathics/format/asy.py

```diff
@@ -310,7 +311,7 @@ def polygon3dbox(self, **options) -> str:
 add_conversion_fn(Polygon3DBox)
 
 
-def polygonbox(self, **options):
+def polygonbox(self, **options) -> str:
```

Comment:
Routines to draw SVG are no longer here but are in `format/svg.py`

Reply:
Oh, ok I gonna rebase it.

Reply:
rebase doesn't track refactoring like this.  You will probably have to move it by hand.